### DISCUSSION TITLE
Why do we need ToastProvider when we already have global Toast

### DISCUSSION THREAD

**Discussion by yongqianvip** [arnnis/react-native-fast-toast#105] [Q&A] [answered]
We can call toast outside React components with `global Toast`, so why do we need `ToastProvider` ? 

Is there anthing that global Toast can not do but `ToastProvider` can ？

**Comment by arnnis** [comment, marked as answer]
ToastProvider is only meant to be an alternative for users who prefer hooks. It also has better typescript. 

You can only use the global toast if you prefer, there is no difference
